Sources/SwifterSwift/SwiftStdlib/CollectionExtensions.swift

```diff
@@ -12,6 +12,16 @@ import Dispatch
 
 // MARK: - Methods
 public extension Collection {
+    
+    /// SwifterSwift: Check if the given collection is not empty.
+    ///
+    ///     [1, 2, 3, 4, 5].isNotEmpty -> True
+    ///     [].isNotEmpty -> False
+    ///
+    /// - Returns: true if the collection is not empty and false if the collection is empty
+    var isNotEmpty: Bool {
+        return !self.isEmpty
```

Comment:
maybe refactor this to a more Swifty style 😉

```swift
var isNotEmpty: Bool { !isEmpty }
```

Reply:
Great suggestion! Updated in my most recent commit.

Reply:
```swift
let someCollectionIsNotEmpty = !someClass.someProperty.anotherStruct.something.isEmpty;
guard someCollectionIsNotEmpty else { return }
// vs.
guard someClass.someProperty.anotherStruct.something.isNotEmpty else { return }
```